go/ethclient/mgmtcontractlib/input_handler_test.go

```diff
@@ -11,7 +11,7 @@ func TestSerialization(t *testing.T) {
 	rol := datagenerator.RandomRollup()
 
 	serializedRollup := EncodeToString(nodecommon.EncodeRollup(&rol))
-	deserializedRollup := DecodeFromString(serializedRollup)
+	deserializedRollup, _ := DecodeFromString(serializedRollup)
```

Comment:
missed this one

Reply:
I think it's fine to ignore as it's in a test.

Reply:
👍